sys-process/audit/audit-3.0.6.ebuild

```
@@ -182,10 +178,11 @@ pkg_postinst() {
 lockdown_perms() {
 	# Upstream wants these to have restrictive perms.
 	# Should not || die as not all paths may exist.
-	# Flatcar: No lockdown of permissions - it's probably only
-	# related to auditd.
-	# local basedir="${1}"
+	# Flatcar: We don't include ausearch and aureport
+	# so they're removed from the hardening list
+	local basedir="${1}"
 	# chmod 0750 "${basedir}"/sbin/au{ditctl,ditd,report,search,trace} 2>/dev/null
+	chmod 0750 "${basedir}"/sbin/au{ditctl,ditd,trace} 2>/dev/null
 	# chmod 0750 "${basedir}"/var/log/audit 2>/dev/null
```

```suggestion
	chmod 0750 "${basedir}"/var/log/audit 2>/dev/null
```
Let's enable this here and check the result of the test build